src/app/component/Header/Header.style.scss

```diff
@@ -322,14 +322,19 @@
             }
 
             &_edit {
-                width: 70px;
```

Comment:
button was not visible, but still had width and moved other objects on mobile in header